.changes/send-params.md

```diff
@@ -0,0 +1,16 @@
+---
+"iota-sdk-bindings-core": patch
+"iota-sdk-nodejs": patch
+"iota-sdk-python": patch
```

Comment:
We only need this for the wallet nodejs binding we release with covector
```suggestion
"wallet-nodejs-binding": patch
```

Reply:
Can you explain?

Reply:
The .change files are used for covector so it knows what change should lead to which version bump and also for the changelog entry
But supported packages need to be configured here https://github.com/iotaledger/iota-sdk/blob/develop/.changes/config.json and we only have the wallet nodejs binding there (we want to get rid of covector alltogether very soon, then we will just do the changelogs as for the Rust crate and don't need the .change files anymore)

Reply:
Oh. Must have changed since I last looked at it. Thanks.